datasets/universal_dependencies/README.md

```diff
@@ -0,0 +1,240 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- af
```

Comment:
In general when the configurations have different languages we use a dictionary config_name -> list of languages like
```
languages:
- config1:
  - lang1
- config2:
  - lang2
```

but given the amount of work that requires and given that you already did an incredible job I guess we can keep that for later if you want x)

Reply:
I will do it! wait for next commit :)

Reply:
Does it looks better?

Reply:
thanks !